facdb/sql/_create_facdb.sql

```diff
@@ -2,7 +2,7 @@ DROP TABLE IF EXISTS facdb;
 SELECT
     TRIM(UPPER(facdb_base.facname)) as facname,
     facdb_address.addressnum,
-    facdb_address.streetname,
+    TRIM(regexp_replace(facdb_address.streetname, '\s+', ' ', 'g')) as streetname,
```

Comment:
this should be done within the facdb_address table 
https://github.com/NYCPlanning/db-facilities/blob/develop/facdb/sql/_create_facdb_address.sql
we want to make sure all the logic/cleaning happens in the same place